.github/workflows/publish_package.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish package
+
+on:
+  push:
+    branches:
+      - master
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to publish"
+        required: false
+        default: ""
+
+jobs:
+  publish-package:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.11
```

Comment:
I assume we should use a different python-version which is compatible with wapi-python

Reply:
https://github.com/volueinsight/wapi-python/actions/runs/7192272678/job/19588399327

I think this Py version is to setup the server for running the packaging work.

Reply:
Great, then it is ok